packages/data-store/src/types/presentationDefinition/presentationDefinition.ts

```diff
@@ -8,10 +9,11 @@ export type PresentationDefinitionItem = {
   name?: string
   purpose?: string
   definitionPayload: IPresentationDefinition
+  dcqlPayload?: DcqlQuery
   createdAt: Date
   lastUpdatedAt: Date
 }
 
 export type NonPersistedPresentationDefinitionItem = Omit<PresentationDefinitionItem, 'id' | 'createdAt' | 'lastUpdatedAt'>
 export type PartialPresentationDefinitionItem = Partial<PresentationDefinitionItem>
-export type PresentationDefinitionItemFilter = Partial<Omit<PresentationDefinitionItem, 'definitionPayload'>> // TODO add logic to linearize & hash definitionPayload into a separate column so we can filter on it?
+export type PresentationDefinitionItemFilter = Partial<Omit<PresentationDefinitionItem, 'definitionPayload' | 'dcqlPayload'>> // TODO add logic to linearize & hash definitionPayload into a separate column so we can filter on it?
```

Comment:
can we move this todo to a ticket and link it on the comment? it can easily be lost on the code

Reply:
Actually I changed my mind and removed the TODO. The same presentation definition payload can be used for multiple configurations and retrieving records by payload hash may not return the intended record or multiple records, so better to avoid it. The caller just needs to the either the unique ID or definitionId + tenantId + version to query the correct record.